plugins/table.md

```diff
@@ -81,8 +81,11 @@ tinymce.init({
 
 ### `table_grid`
 
-This option allows you disable the table grid feature of the table menu. The grid feature (which is enabled by default) consists of a dragable matrix that a user can interact with to automatically create a table of x-rows by x-columns based upon their input.
-However, if you set this option to `false` the dragable matrix will be replaced with a table dialog box into which a user can enter various parameters to create a table cols, rows, width, height, cell spacing, cell padding, border, caption and alignment). There are also advanced style options available in the dialog.
+This option allows you to disable the grid-like table picker in the Table menu. The grid feature (which is enabled by default) consists of a dragable matrix that a user can interact with to automatically create a table of x-rows by x-columns based upon their input.
+
+However, if `table_grid` is set to `false` the table picker will be replaced by a menu item that opens a dialog that users can use to insert a dialog. This dialog allows users to set various parameters such as the number of columns and rows, width, height, cell spacing and padding, border width, alignment, and whether to display a caption. There are also advanced style options available if `table_advtab` is set to `true`.
```

Comment:
opens a dialog that users can use to insert a dialog.
change to 
opens a dialog that users can use to insert a table.

Reply:
done, thanks!